crates/metaslang/cst/src/text_index.rs

```diff
@@ -114,7 +121,7 @@ impl TextRangeExtensions for TextRange {
         self.start.utf16..self.end.utf16
     }
 
-    fn char(&self) -> Range<usize> {
-        self.start.char..self.end.char
+    fn line(&self) -> Range<usize> {
```

Comment:
what do you think of using `lines()` instead of `line()` for the public API?
since it is a range of lines.

```suggestion
    fn lines(&self) -> Range<usize>;
```